### ISSUE TITLE
Ajax loaded tabs

### ISSUE THREAD

**Issue by dimdimich** [tmhub/easytabs#17]
Does it now dynamically update the content for simple products within a configurable product (i.e. having a specification tab, with different content of each simple product)?

**Comment by dimdimich** [comment]
What do you think about adding such feature to main branch or it should be paid modification?

**Comment by tangar76** [comment]
I do not think such feature should be implemented in easytabs. It has to be some custom development - some block that can load data via ajax from simple product on config option change. Then this block can be place into tabs via Magento admin interface.

**Comment by dimdimich** [comment]
I think that it's quite popular way to load content on content heavy pages. Please discuss that with Vova.

**Comment by vovayatsyuk** [comment]
TL;DR;
It's possible but unstable. Especially regarding third-party tab contents.
So, if we will add this feature, there should be an easy way to disable it per tab.

**Explantation**

- For now, tabs are rendered during `catalog_product_view` action with access to Request object (all params), Registry, Events.
- If we implement some action to render tab, the tab will be rendered during `easytabs_tab_render` action with different environment

This may cause errors for some the tabs that require some logic that works during `catalog_product_view` action only.

Additionally, various js may not work as expected (dom:loaded listener for example).

**Comment by vovayatsyuk** [comment]
Another caveat:

Customers actively use our feature, when the tab is not rendered if there is no content for it. 

This feature will not work for ajax tabs, of course.
